com.microsoft.mrtk.graphicstools.unity/Editor/Inspectors/RectMask2DFastInspector.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.MixedReality.GraphicsTools.Editor
     [CanEditMultipleObjects]
     public class RectMask2DFastInspector : RectMask2DEditor
     {
-        [MenuItem("GameObject/UI/Rect Mask Fast - Graphics Tools")]
+        [MenuItem("GameObject/UI/Graphics Tools/Rect Mask Fast")]

```

Comment:
Also should this be renamed Fast Rect Mask? To keep in line with the naming of the Rounded Rect Mask?

Reply:
Good idea!